### PR TITLE
Building a Loader Using Animations in Tailwind CSS

### LINKED ISSUE #2468: Building a loader using animations in Tailwind CSS.
We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner. Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

### Proposed title of article
Building a loader using animations in Tailwind CSS.

### Introduction paragraph (2-3 paragraphs):
What you see on the screen while the rest of the page’s content is still loading is what is called loaders/preloaders.  They are a fun way to bring life into a webpage. Loaders can be simple or complex animations that may keep site visitors entertained as they wait for the servers to load the page content.
Although there are some websites that offer free loaders as GIFs, we could build our own! After all we are developers, right?

In this tutorial , you are going to be taken through the various Tailwind CSS classes , along with some custom CSS  used in building beautiful and fun loaders.

### Key takeaways: 
1. Readers will learn how to use the animation classes in Tailwind CSS and animation elements in custom CSS.
2. Readers will understand how loaders can improve user experience.
3. Readers will understand the importance of using simple loaders for their applications.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by M0nicah**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

This fixes #2468

### CODE REVIEW

**Review by kayere — APPROVED** [review]
Awesome article @M0nicah, keep up the good work.

**Comment by louisefindlay23** [comment]
@hectorkambow, can you please run a plagiarism check on this article?

**Comment by hectorkambow** [comment]
plagiarism check done

File: content/articles/building-a-loader-using-animations-in-tailwind-css/index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		animation-delay: 0.3s;
+	}
+</style>
+```
+
+The above code has added time for which each circle would wait before it bounces. For instance, the red circle waits for 0.3 seconds before it bounces. It waits for the green circle to bounce before it does. The green circle on the other hand waits for the blue circle to bounce before it does
+
+Here's our very simple loader in just a few, simple steps!
+
+![Animated Loader](video.gif)
```
Comment (LinusMuema) [inline content/articles/building-a-loader-using-animations-in-tailwind-css/index.md:159]: @M0nicah 
I believe the referenced resource at this point is missing in the article folder. Could you please check on it.

cc: @kayere

**Review by LinusMuema — CHANGES_REQUESTED** [review]

**Review by LinusMuema — APPROVED** [review]

**Comment by hectorkambow** [comment]
Great work - very nice tutorial - very easy to follow along @M0nicah 

cc @kayere 👍 

cc @LinusMuema  

Please be sure to go through my final edits to better improve future articles/reviews.

**Pull request merged**